.github/workflows/source_k8s.yml

```diff
@@ -70,9 +70,6 @@ jobs:
       - name: Fail if docs are changed
         if: github.event_name == 'pull_request'
         run: test "$(git status -s ./docs/tables | wc -l)" -eq 0
-      - name: gen-code
-        if: github.event_name == 'pull_request'
-        run: make gen-code
       - name: Fail if code generation changed services
         if: github.event_name == 'pull_request'
         run: test "$(git status -s ./resources/services | wc -l)" -eq 0
```

Comment:
Rather than remove `gen-code`, I think this should be:

```
      - name: gen
        if: github.event_name == 'pull_request'
        run: make gen
      - name: Fail if generation updated files
        if: github.event_name == 'pull_request'
        run: test "$(git status -s | wc -l)" -eq 0 || (git status -s; exit 1)
```